views/dpsSearchResults.njk

```diff
@@ -22,10 +22,11 @@
   <div class="govuk-grid-row">
     <h1 class="govuk-heading-l">{{ title }}</h1>
 
+    {% block searchFilter %}
```

Comment:
show the filtering irrespective of whether any search results, since could be no search results as result of filtering choices